app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -442,30 +436,40 @@ class _ReplyContentWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (msg is TextMessage) {
-      return messageWidget!;
-    } else if (msg is ImageMessage) {
+    if (msg is ImageMessage) {
+      var imageMsg = msg as ImageMessage;
       return Padding(
         padding: const EdgeInsets.all(8.0),
-        child: ConstrainedBox(
-          constraints: const BoxConstraints(maxHeight: 100, maxWidth: 125),
-          child: ClipRRect(
-            borderRadius: BorderRadius.circular(6.33),
-            child: Image.memory(
-              base64Decode(msg?.metadata?['base64']),
-              fit: BoxFit.fill,
-              cacheWidth: 125,
-            ),
-          ),
+        child: ImageMessageBuilder(
+          message: imageMsg,
+          messageWidth: imageMsg.size.toInt(),
+          isReplyContent: true,
         ),
       );
-    } else if (msg is FileMessage) {
-      return messageWidget!;
-    } else if (msg is CustomMessage) {
-      return messageWidget!;
-    } else {
-      return const SizedBox.shrink();
+    } else if (msg is TextMessage) {
+      var textMsg = msg as TextMessage;
+      bool enlargeEmoji = false;
+      if (textMsg.metadata!.containsKey('enlargeEmoji')) {
+        enlargeEmoji = textMsg.metadata!['enlargeEmoji'];
+      }
+      return Container(
+        alignment: Alignment.center,
+        constraints: const BoxConstraints(maxHeight: 120),
+        padding: const EdgeInsets.all(8),
+        child: enlargeEmoji
+            ? Text(textMsg.text, maxLines: 3, overflow: TextOverflow.ellipsis)
```

Comment:
why does it depend on whether we show emoji to show the HTML version? Shouldn't we always show the HTML version if given?

(sidenote that we should make clear if there is an HTML version or only the blank text version, but that is a Rust-API fix to do at another time)

Reply:
Yes, we should always show HTML version in reply content UI.